src/pyOpenMS/tests/unittests/test_core_structures.py

```
@@ -0,0 +1,768 @@
+    assert trace.setCentroidSD is not None
+    assert trace.setSmoothedIntensities is not None
+    assert trace.updateSmoothedMaxRT is not None
+    assert trace.updateWeightedMeanRT is not None
+    assert trace.updateSmoothedWeightedMeanRT is not None
+    assert trace.updateMedianRT is not None
+    assert trace.updateMedianMZ is not None
+    assert trace.updateMeanMZ is not None
+    assert trace.updateWeightedMeanMZ is not None
+    assert trace.updateWeightedMZsd is not None
+
+    s = trace.getSize()
```

Ruff (0.14.10)

Local variable `s` is assigned to but never used

Remove assignment to unused variable `s`

(F841)